selfservice/flow/login/extension_identifier_label.go

```diff
@@ -6,32 +6,57 @@ package login
 import (
 	"context"
 
+	"github.com/pkg/errors"
+	"github.com/samber/lo"
+
+	"github.com/ory/herodot"
 	"github.com/ory/kratos/text"
 
 	"github.com/ory/jsonschema/v3"
 	"github.com/ory/kratos/schema"
 )
 
 type identifierLabelExtension struct {
+	field                     string
 	identifierLabelCandidates []string
 }
 
-var _ schema.CompileExtension = new(identifierLabelExtension)
+var (
+	_               schema.CompileExtension = new(identifierLabelExtension)
+	ErrUnknownTrait                         = herodot.ErrBadRequest.WithReasonf("Trait does not exist in identity schema")
```

Comment:
Is that something the requester (user) can fix, or is this more a server misconfiguration (500)?